test/tornado_tests/test_motor_client.py

```diff
@@ -307,6 +308,40 @@ async def test_handshake(self):
         except Exception:
             pass
 
+    @gen_test
+    async def test_driver_info(self):
+        server = self.server()
+        driver_info = DriverInfo(name="Foo", version="1.1.1", platform="FooPlat")
+        client = motor.MotorClient(server.uri, driver=driver_info)
+
+        # Trigger connection.
+        future = client.db.command("ping")
```

Comment:
Is this needed? IIUC the ismaster is from SDAM which happens automatically so this future could be removed.

Reply:
Ah, I wasn't sure if it was needed. I assumed that the ping was the source.

Reply:
Yeah, I tried removing the ping call, and the test now crashes with `asyncio.exceptions.TimeoutError: Operation timed out after 5 seconds`